README.md

```diff
@@ -2,33 +2,47 @@
-  - Analyze and validate JSON syntax, JSON schema conformance, and string expressions.
+- [Snippets](#Snippets)
+- [Azure schema integration](#Azure-schema-completion-and-validation)
+- [Parameter file support](#parameter-files)
+- [Completions](#completions)
+- [Template navigation](#template-navigation)
+- [Other features](#other-features)
+- [Extension configuration](#extension-configuration)
 
-## Parameter files
+## Snippets
 
-Template files can be mapped to a parameter file for an enhanced authoring and validation experience. To do so use the parameter file control in the Visual Studio Code status bar, or use the right-click contextual menu on an ARM template. Once a parameter file has been mapped, the following features are available.
+Snippets are used to create basic templates and add elements such as parameters, resources, and outputs to existing templates.
 
-  - Full validation across both the template and parameter file
-  - Open an associated parameter file from a template
-  - Add all missing parameters to the parameter file and parameter auto-completion
-  - Rename parameters and all references across both the template and mapped parameter file
-  - Find and goto all references of a parameter across both the template and mapped parameter file
+### Create a new ARM template
 
-![Associate a parameters file with template and create parameters file](./images/params-support.gif)
+When in an empty JSON file, typing `arm` produces a list of scaffolding snippets. These snippets can be used to create an empty template for any ARM deployment scope (Tenant, Subscription, Management Group, and Resource Group).
 
-## Snippets
+![Image showing the arm template scaffolding snippet](./images/json-snippets.png)
+
+Notice that adding scaffolding for an ARM template changes the Visual Studio Code language mode to *Azure Resource Manager Templates*.
+
+![Image showing that the language type has changed to Azure Resource Manager Template](./images/arm-template-language.png)
+
+### Add Azure resource to an ARM template
 
-Snippets for deployment template scaffolding and 70+ Azure resources.
+Once you are working in an ARM template, placing your cursor in the resource and typing `arm` produces a list of 70+ snippets for Azure resources. Selecting a snippet adds an instance of the resource type to the ARM template.
+
+![Image showing a storage account being added with a snippet](./images/arm-snippets.png)
+
+Each snippet includes tab stops to help navigate through commonly modified configurations. Press the `tab` key to navigate to the next stop, and `shift` + `tab` to navigate to the previous stop. As seen in the following example, some stops are pre-populated with a list of common values for quick selection.
+
+![Image showing snippet tab stops with pre-populated parameter values](./images/tab-stops.png)
+
+### Snippet Reference
+
+The following table lists the available ARM Tools snippets.
 
 | Snippet | Description |
 |---|---|
```

Comment:
Needs updating

Reply:
This is updated, please add any feedback / suggested changes.